doc/tor.md

```diff
@@ -211,3 +257,13 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- If you are in an environment that does not permit direct Tor connections,
+  and does not permit the public bridges, then considering the trade-offs
+  it probably isn't safe enough to use the Tor network. Example: few places
+  in China and India
+
+- If someone is operating in paranoid mode then the correct move is to not
+  listen at all on the node they are trying to protect, because any listening
+  lets attackers actively connect into them. Frequently changing local onion
+  address will not be helpful.
```

Comment:
I'm not sure about this content, will let other reviewers weigh in.

Reply:
This was mentioned by Greg Maxwell when we were discussing about adding an option that deletes "onion private key" regularly: https://github.com/bitcoin/bitcoin/issues/17491#issuecomment-705485718

Reply:
I agree with Greg; am less sure about the rest of the two paragraphs and the writing could be improved, but better for others to weigh in.

Reply:
I think this is formulated strangely. Sure, frequently changing onion addresses is silly, but nothing in this document is suggesting doing that. You're also not defining what "paranoid mode" means.

What about this instead?

> For maximal privacy, not listening at all is preferable over creating a hidden service.

Reply:
Done. Made changes in https://github.com/bitcoin/bitcoin/pull/21157/commits/46329146b3c111490960bf4072c83b3f9015a9c1 and replaced with:

_For maximal privacy, not listening at all is preferable over creating an onion service._

Since we have not mentioned about deleting "Onion Private Key" in this doc, maybe its best to edit the Bitcoin Wiki to remove or change that line: https://en.bitcoin.it/w/index.php?title=Setting_up_a_Tor_hidden_service